internal/eval/eval.go

```diff
@@ -54,6 +54,22 @@ type EvalResponse struct {
 	Error  error       // Error holds any error that occurred during the operation. If no error, it will be nil.
 }
 
+//go:inline
+func makeEvalResult(result interface{}) *EvalResponse {
```

Comment:
Can we have a comment about this function?

Reply:
Hi @AshwinKul28 , Please check the #1186. I merged the PR eagerly I suppose. Apologies. We can use that PR for more updates If left in this PR.